openml-lightgbm/lightgbm-builder/pom.xml

```diff
@@ -187,20 +106,17 @@
                 </executions>
             </plugin>
 
-
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>2.7</version>
+                <configuration>
+                    <!-- skip deployment because lightgbm-lib is already included in uber.jar -->
```

Comment:
not `lightgbm-lib`, now it's named `lightgbm-builder`

Reply:
and replace `uber.jar` by `uber jar`

Reply:
Although the name of folder is lightgbm-builder, the artifact is still lightgbm-lib, I think we should keep it as lightgbm-lib :thinking: 
I will replace uber.jar to uber jar